l4ping/l4ping.TESTGEN.ps1

```diff
@@ -0,0 +1,143 @@
+#===============================================
+# Script Input Parameters Enforcement
+#===============================================
+Param(
+    [parameter(Mandatory=$false)] [string] $Config = "Default",
+    [parameter(Mandatory=$true)]  [string] $DestIp,
+    [parameter(Mandatory=$true)]  [string] $SrcIp,
+    [parameter(Mandatory=$true)]  [ValidateScript({Test-Path $_ -PathType Container})] [String] $OutDir = "" 
```

Comment:
nit: single quotes

Reply:
Sure, I can do that. I copied this from the way all the other TestGen scripts are written. Would you like me to change all of them?

Reply:
Removed the default value here.